src/patternfly/components/Login/docs/code.md

```diff
@@ -2,21 +2,23 @@
 
 | Class | Applied To | Outcome |
 | -- | -- | -- |
-| `.pf-c-login` | `<div>` |  Initializes the login component. |
-| `.pf-c-login__container` | `<div>` |  Positions the login component content. |
-| `.pf-c-login__header` | `<header>` |  Positions the login header. |
+| `.pf-c-login` | `<div>` |  Initializes the login component. **Required**|
+| `.pf-c-login__container` | `<div>` |  Positions the login component content. **Required**|
+| `.pf-c-login__header` | `<header>` |  Positions the login header. **Required**|
 | `.pf-c-login__header .pf-c-brand` | `<img>` |  Creates a brand image inside of login header. |
```

Comment:
Currently the margin between the header and login box comes from `.pf-c-login__header .pf-c-brand`, but if the brand isn't required, what's acceptable to put in the header?

Reply:
If the header is required, maybe we should move the margin from the brand to `.pf-c-login__header`?

Reply:
IMO this gives no flexibility for the user to add different content other than an image. But I'm fine to make it required for now, since its dictating space.

Reply:
Cool. I'm not sure if we want someone to have anything other than an image in there? If we want to support text, along with moving the margin for the header from the image to the header itself, we might also want to recommend what size title component to use in that space.